packages/input-date/src/LionInputDate.js

```diff
@@ -34,6 +34,8 @@ export class LionInputDate extends LocalizeMixin(LionInput) {
     this.parser = value => (value === '' ? undefined : parseDate(value));
     this.formatter = formatDate;
     this.defaultValidators.push(new IsDate());
+    // Just explicitly make clear we shouldn't use type 'date'
+    this.type = 'text';
```

Comment:
When it was set in connectedCallback, initial parsers / serializers would not get the right value (because they run before connectedC.)